scripts/check-pkg-for-release.js

```diff
@@ -278,3 +319,19 @@ function getPkgName() {
   }
   return name;
 }
+
+function getMissingDependencyRemedy(missingDeps) {
+  let remedyStr = 'Please add the missing dependencies using the following command(s):\n';
+  missingDeps.forEach((dep) => {
+    remedyStr += `npx lerna add ${dep} packages/${PACKAGE_RELATIVE_PATH.split('/')[1]}\n`;
```

Comment:
This suggestion might mislead someone who exported a CL, executing this command would only the PR. Can we also list out missing dependency (with version number) so that we can manually copy & paste it to package.json internally?

WDYT?

Reply:
How about checking if this is an exported CL by seeing if branch is `cl_*`, then conditionally printing a warning message following this to patch all `package.json` changes to internal? This saves us from having to dig up the version number using something like `npm show {pkg} version`, which learna would have done anyways

Reply:
Can we show both suggestions? Author will figure out how to fix it.

For example,

```
Failure: Missing dependencies in mdc-checkbox
  @material/feature-targeting@4.0.0, @material/density@4.0.0

Manually add these dependencies to mdc-checkbox's package.json file
or run following command on root of GitHub repository for auto fix.
  npx lerna add <missing_deps> ...
```

WDYT?

Reply:
For versioning, is it safe to assume master `material-components-web` version will always be the same as the sub-module versions? 4.0.0

Reply:
Version of material-components-web and sub-modules are not always same.

But we can get a current version of a package by executing this command:

```shell
npm show @material/checkbox version
```

Reply:
I revised the script to include these details in the message and accommodate the new sass module changes